sdk/template/Azure.Template/tests/MiniSecretClientLiveTests.cs

```diff
@@ -8,18 +8,26 @@
 
 namespace Azure.Template.Tests
 {
+    [ClientTestFixture(
+        // List all supported service versions below.
+        MiniSecretClientOptions.ServiceVersion.V7_0
+    )]
     public class MiniSecretClientLiveTests: RecordedTestBase<MiniSecretClientTestEnvironment>
     {
-        public MiniSecretClientLiveTests(bool isAsync) : base(isAsync)
+        private readonly MiniSecretClientOptions.ServiceVersion _serviceVersion;
+
+        public MiniSecretClientLiveTests(bool isAsync, MiniSecretClientOptions.ServiceVersion serviceVersion)
+            : base(isAsync, null /* RecordedTestMode.Record /* delete "null /* " to re-record */)
```

Comment:
We should recomend https://github.com/Azure/azure-sdk-for-net/blob/main/eng/nunit.runsettings#L18 instead. Or jus the RecordedTest attribute

Reply:
![image](https://user-images.githubusercontent.com/1279263/150237350-221d41ce-f95f-4d8f-ad71-e90dc84e9fdb.png)

Reply:
We still need an explicit way to re-record.

Reply:
I believe you can still set the mode to Record

Reply:
> We should recomend https://github.com/Azure/azure-sdk-for-net/blob/main/eng/nunit.runsettings#L18 instead. Or jus the RecordedTest attribute

I wasn't even aware of that. Why not use something locally? Quite a few services have been copying this pattern courtesy of @tg-msft's influence.

> We still need an explicit way to re-record.

That's precisely why I do this. Besides, at least at one point, `[RecordedTest]` (which the template does use) didn't work for new tests i.e., it expected a recorded session file had to already exist. Perhaps that's been fixed, but this mode can also be changed to `Live` to just debug a live run.

Reply:
I could update the comment to say that tests are re-recorded automatically, but this is useful for forcing it or running live, etc. How does that sound instead?

Reply:
>Why not use something locally? 

What do you mean by locally?

Reply:
FWIW, even with `[RecordedTest]` there's been a couple times with Cognitive where I still needed to forcefully re-record because the service behavior actually changed a little such that the code + recording wasn't wrong, so no re-recording was triggered. Rare, perhaps, but still possible.

Reply:
> > Why not use something locally?
> 
> What do you mean by locally?

If I want to change the mode and record or debug live when I'm already in a class, I just delete the `null /* ` text vs. I have to remember where that nunit file is and change that line accordingly. It's...extra.

Reply:
Good thread for context https://github.com/Azure/azure-sdk-for-net/issues/14176

Reply:
Just added a comment. You can practically just swap environment variable values. Way too many subprocesses which don't automagically pick up new env vars from their parent processes (environment variables are copied to subprocesses invariably across platforms).

I don't think we get rid of the nunit file, I'm just in favor - like some others in that thread - of providing a class-local way of overriding as needed. It's right there in front of your face and we don't need to educate users on some other file elsewhere in the repo that's not part of the VS solution, nor probably whatever workspace devs have open in VSCode.

Reply:
As discussed offline, I added a comment about this and nunit.runsettings.